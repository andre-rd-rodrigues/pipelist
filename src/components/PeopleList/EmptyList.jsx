import React from "react";
import { motion } from "framer-motion";
import {
  containerVariant,
  horizontalEntranceLeft
} from "styles/motion/motionVariants";
import styles from "./list.module.scss";

function EmptyList() {
  return (
    <motion.div
      variants={containerVariant}
      initial="hidden"
      animate="visible"
      className={styles.emptyList}
    >
      <motion.img
        variants={horizontalEntranceLeft}
        src={`${process.env.PUBLIC_URL}/images/start.png`}
        alt="Start your pipelist!"
      />
      <motion.p variants={horizontalEntranceLeft}>
        Currently your list is empty. <b>Add a person</b> to start your very
        first pipelist!
      </motion.p>
    </motion.div>
  );
}

export default EmptyList;
