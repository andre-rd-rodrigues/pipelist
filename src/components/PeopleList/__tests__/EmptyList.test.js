import reactTestRenderer from "react-test-renderer";
import EmptyList from "../EmptyList";

describe("Empty list", () => {
  it("should render correctly", () => {
    const wrapper = reactTestRenderer.create(<EmptyList />);

    expect(wrapper).toMatchSnapshot();
  });
});
