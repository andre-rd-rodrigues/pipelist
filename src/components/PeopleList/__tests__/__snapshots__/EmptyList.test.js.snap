// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty list should render correctly 1`] = `
<div
  className="emptyList"
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <img
    alt="Start your pipelist!"
    src="/images/start.png"
    style={
      Object {
        "opacity": 0,
        "transform": "translateX(-90px) translateZ(0)",
      }
    }
  />
  <p
    style={
      Object {
        "opacity": 0,
        "transform": "translateX(-90px) translateZ(0)",
      }
    }
  >
    Currently your list is empty. 
    <b>
      Add a person
    </b>
     to start your very first pipelist!
  </p>
</div>
`;
